pepper-apis/src/main/java/org/broadinstitute/ddp/client/SendGridClient.java

```diff
@@ -76,6 +77,7 @@ public SendGridClient(String apiKey, @Nullable String proxy) {
                 throw new IllegalArgumentException("proxy needs to be a valid url");
             }
             httpClientBuilder.setProxy(new HttpHost(proxyUrl.getHost(), proxyUrl.getPort(), proxyUrl.getProtocol()));
+            httpClientBuilder.setConnectionReuseStrategy(NoConnectionReuseStrategy.INSTANCE);
```

Comment:
I have seen some of the SendGrid requests fail with connection issue, especially the request to get template version. I am less certain about this change here, even less than the change to the Elasticsearch HTTP client. However, I think the SendGrid client is in the same situation, so not reusing connections to proxy should hopefully help.